src/main/java/com/gameshop/Controller/OrderController.java

```diff
@@ -0,0 +1,70 @@
+package com.gameshop.Controller;
+
+import com.gameshop.dto.OrderDto;
+import com.gameshop.entity.Order;
+import com.gameshop.entity.UserInfo;
+import com.gameshop.exception.ResourceNotFoundException;
+import com.gameshop.repository.OrderRepo;
+import com.gameshop.repository.UserRepo;
+import com.gameshop.service.OrderService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import javax.xml.ws.Response;
+import java.util.List;
+
+@RestController
+@RequestMapping("/order")
+public class OrderController {
+
+    @Autowired
+    OrderRepo orderRepo;
+    @Autowired
+    UserRepo userRepo;
+    @Autowired
+    OrderService orderService;
+
```

Comment:
Use constructor Autowiring - this is Spring recommendation

And this will reduce the amount of code